backend-rust/README.md

```diff
@@ -141,41 +164,42 @@ Given that one wants follow the logs of the database:
 docker logs -f postgres_db
 ```
 
-### Database migrations
+### Running database migrations
 
-Database migrations are tracked in the `migrations` directory.
-To introduce a new one run:
+To setup the database schema when developing the service run:
 
 ```
-sqlx migrate add '<description>'
+cargo run --bin ccdscan-indexer -- --migrate-only
 ```
 
-where `<description>` is replaced by a short description of the nature of the migration.
+NOTE: Having compile-time checked queries will cause issues, since the queries are invalid until the database have been properly migrated. This is done by _not_ having the `DATABASE_URL` environment variable set until after running the migrations or using:
```

Comment:
This `Note` could be added to the ` ccdscan-api` service further up as well with the blow cli command as a work around since it can happen in the  ` ccdscan-api` service as well.